src/coins/coins.ts

```diff
@@ -208,12 +212,7 @@ export const basicCoins: BasicCoin[] = [
       [ChainId.POL]: {
         address: '0xa649325aa7c5093d12d6f98eb4378deae68ce23f',
         decimals: 18,
-      },
-      [ChainId.DAI]: {
-        address: '0xca8d20f3e0144a72c6b5d576e9bd3fd8557e2b04',
-        decimals: 18,
-        symbol: 'WBNB',
-        name: 'Wrapped BNB',
+        name: 'Multichain: BNB Token',
```

Comment:
https://gopluslabs.io/token-security/137/0xa649325aa7c5093d12d6f98eb4378deae68ce23f
this token looks weird...

Reply:
thanks for your checking, gonna remove
I got it from Coinmarcketcap